gbpservice/tests/contrib/nfp_scripts/upgrade_nfp.sh

```diff
@@ -0,0 +1,146 @@
+#! /bin/bash
+
+source upgrade_nfp.conf
+source $DEVSTACK_DIR/local.conf
+
+#TODO(DEEPAK): Should be retrieved from a result file populated by advanced mode.
+EXT_NET_NAME=ext-net
+
+function create_port_for_vm {
```

Comment:
This function is already available in one of our devstack scripts right? Is there a possibility of reusing it by sourcing the script?